assets/js/state/hosts.js

```diff
@@ -59,6 +59,25 @@ export const hostsListSlice = createSlice({
         return host;
       });
     },
+    setHostListDeregisterable: (state, { payload }) => {
+      const ids = payload.map((host) => host.id);
+
+      state.hosts = state.hosts.map((host) => {
+        if (ids.includes(host.id)) {
+          host.deregisterable = true;
```

Comment:
Just a very nitpick comment, it's true that redux toolkit abstract the immutability for us, so we can assign to state properties without worring about immutability (using immer under the hood https://immerjs.github.io/immer/), but, we are mapping on an array, and it's good practice to not reassign the paramters like here, if you want you can change this condition to `return {...host, deregisterable: true}` and the full `{...host}` after.